packages/utils/upgrade/src/cli/index.ts

```diff
@@ -1,32 +1,62 @@
 import os from 'os';
 import chalk from 'chalk';
-import { program } from 'commander';
+import { Option, program } from 'commander';
 
 import { version as packageJSONVersion } from '../../package.json';
 import { Version } from '../modules/version';
 
 import type { CLIOptions } from './types';
 
+const projectPathOption = new Option(
+  '-p, --project-path <project-path>',
+  'Path to the Strapi project'
+);
+
+const dryOption = new Option(
+  '-n, --dry',
+  'Simulate the upgrade without updating any files'
+).default(false);
+
+const debugOption = new Option('-d, --debug', 'Get more logs in debug mode').default(false);
+
+const silentOption = new Option('-s, --silent', "Don't log anything").default(false);
+
+const automaticConfirmationOption = new Option(
+  '-y, --yes',
+  'Automatically answer "yes" to any prompts that the CLI might print on the command line.'
+).default(false);
+
 const addReleaseUpgradeCommand = (releaseType: Version.ReleaseType, description: string) => {
   program
     .command(releaseType)
     .description(description)
-    .option('-p, --project-path <project-path>', 'Path to the Strapi project')
-    .option('-n, --dry', 'Simulate the upgrade without updating any files', false)
-    .option('-d, --debug', 'Get more logs in debug mode', false)
-    .option('-s, --silent', "Don't log anything", false)
-    .option(
-      '-y, --yes',
-      'Automatically answer "yes" to any prompts that the CLI might print on the command line.',
-      false
-    )
+    .addOption(projectPathOption)
+    .addOption(dryOption)
+    .addOption(debugOption)
+    .addOption(silentOption)
+    .addOption(automaticConfirmationOption)
     .action(async (options: CLIOptions) => {
       const { upgrade } = await import('./commands/upgrade.js');
 
       return upgrade({ ...options, target: releaseType });
     });
 };
 
+program
+  .command('codemods')
+  .description(
+    'Run the upgrade process with the selected codemods without updating the Strapi dependencies'
```

Comment:
We're not really running the upgrade process though, are we? Could we simplify it to say it'll simply only run codemods for the desired target or smth like that?